jetbrains: 2025.2.4 -> 2025.3.0.3

jamesward opened the pull request:
jetbrains.clion: 2025.2.4 -> 2025.2.5
    jetbrains.datagrip: 2025.2.4 -> 2025.3
    jetbrains.gateway: 2025.2.4 -> 2025.2.5
    jetbrains.goland: 2025.2.4 -> 2025.2.5
    jetbrains.idea-community: 2025.2.4 -> 2025.2.5
    jetbrains.idea-ultimate: 2025.2.4 -> 2025.2.5
    jetbrains.phpstorm: 2025.2.4 -> 2025.2.5
    jetbrains.pycharm-community: 2025.2.4 -> 2025.2.5
    jetbrains.pycharm-professional: 2025.2.4 -> 2025.2.5
    jetbrains.rider: 2025.3 -> 2025.3.0.3
    jetbrains.ruby-mine: 2025.2.4 -> 2025.2.5
    jetbrains.rust-rover: 2025.2.4.1 -> 2025.2.5
    jetbrains.webstorm: 2025.2.4 -> 2025.2.5

## Things done

- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [x] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [ ] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

theCapypara commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 463570`
Commit: `5fca555fc134cbf63f1a5a4af3589bfd8e9ef01f`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 3 packages built:</summary>
  <ul>
    <li>jetbrains.idea-community-bin</li>
    <li>jetbrains.idea-ultimate</li>
    <li>jetbrains.rider</li>
  </ul>
</details>

theCapypara commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 463570`
Commit: `b9b1cd300c380f4f12c8224374f0aaaf6233dd50`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 13 packages built:</summary>
  <ul>
    <li>jetbrains.clion</li>
    <li>jetbrains.datagrip</li>
    <li>jetbrains.gateway</li>
    <li>jetbrains.goland</li>
    <li>jetbrains.idea-community-bin</li>
    <li>jetbrains.idea-ultimate</li>
    <li>jetbrains.phpstorm</li>
    <li>jetbrains.pycharm-community-bin</li>
    <li>jetbrains.pycharm-professional</li>
    <li>jetbrains.rider</li>
    <li>jetbrains.ruby-mine</li>
    <li>jetbrains.rust-rover</li>
    <li>jetbrains.webstorm</li>
  </ul>
</details>

Jysve reviewed (approved):
If possible, noting that there are some newer versions available:

**jetbrains**: 2025.2.4 -> ~~2025.3.0.3~~ 2025.3.0.4
**jetbrains.dataspell**: 2025.2.3 -> 2025.3
**jetbrains.gateway**: 2025.2.4 -> ~~2025.2.5~~ 2025.3
**jetbrains.rust-rover**: 2025.2.4.1 -> ~~2025.2.5~~ 2025.3
**jetbrains.rider**: 2025.3 -> ~~2025.3.0.3~~ 2025.3.0.4

edwtjo reviewed (commented):
I believe you should squash the Dec 3 commits into the Dec 2 commits, ending up with just 2 for the version bump, this otherwise this looks fine

theCapypara commented:
@jamesward Can you squash? Then this is good to merge :)

jamesward committed:
jetbrains: 2025.2.4 -> 2025.3.0.3

jetbrains.clion: 2025.2.4 -> 2025.2.5
jetbrains.datagrip: 2025.2.4 -> 2025.3
jetbrains.gateway: 2025.2.4 -> 2025.2.5
jetbrains.goland: 2025.2.4 -> 2025.2.5
jetbrains.idea-community: 2025.2.4 -> 2025.2.5
jetbrains.idea-ultimate: 2025.2.4 -> 2025.2.5
jetbrains.phpstorm: 2025.2.4 -> 2025.2.5
jetbrains.pycharm-community: 2025.2.4 -> 2025.2.5
jetbrains.pycharm-professional: 2025.2.4 -> 2025.2.5
jetbrains.rider: 2025.3 -> 2025.3.0.3
jetbrains.ruby-mine: 2025.2.4 -> 2025.2.5
jetbrains.rust-rover: 2025.2.4.1 -> 2025.2.5
jetbrains.webstorm: 2025.2.4 -> 2025.2.5

jamesward committed:
jetbrains.plugins: update

jamesward commented:
@theCapypara Squashed. Thank you :)
